bundles/edu.kit.kastel.sdq.eclipse.grading.core/src/edu/kit/kastel/sdq/eclipse/grading/core/SystemwideController.java

```diff
@@ -112,20 +117,24 @@ private IAssessmentController getAssessmentController(ISubmission submission, IC
 			return List.of();
 		}
 
-		return this.getArtemisGUIController().getBegunSubmissions(this.exercise).stream().filter(submissionFilter).collect(Collectors.toList());

```

Comment:
Formatting. Please apply the formatter on your files (also save actions) .. I won't write that to other occurrences :)